yt_dlp/extractor/gronkh.py

```diff
@@ -48,6 +52,9 @@ def _real_extract(self, url):
                 'url': data_json['vtt_url'],
                 'ext': 'vtt',
             })
+        if data_json.get('chapters'):
+            chapters = list(map(lambda x: {'title': x['title'], 'start_time': x['offset']}, data_json.get('chapters')))
+            chapters[-1]['end_time'] = data_json.get('source_length')
```

Comment:
this can be removed

core code will set the last chapter's end_time from duration